recipes/pycraf-gui/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pycraf-gui" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycraf-gui-{{ version }}.tar.gz
+  sha256: 1403ac72528b2f4600e89d4a1746888162cadc734490d819f2c08c6a4d1a4910
+
+build:
+  entry_points:
+    - pycraf-gui = pycraf_gui.__main__:main
```

Comment:
Only `console_scripts` need to be provided here: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
In my first attempt, I left that out, which had the consequence that on Windows an error was thrown (it created a binary, pycraf-gui.exe, which was considered as not suitable for noarch). The above snippet with the entry_points was actually suggested by the grayskull [webapp](https://www.marcelotrevisani.com/grayskull).

Reply:
Ahh you‘re right - ok then that‘s fine